code/game/turfs/open/glass.dm

```diff
@@ -13,8 +13,11 @@
 	barefootstep = FOOTSTEP_HARD_BAREFOOT
 	clawfootstep = FOOTSTEP_HARD_CLAW
 	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
+	var/stateDamage = "glass"
```

Comment:
```suggestion
	var/state_damage = "glass"
```
Favor snake case and please document new variables.

Reply:
snake case?

Reply:
`snake_case`, `camelCase`, `PascalCase`